cirq-google/cirq_google/api/v2/program.proto

```diff
@@ -617,3 +618,8 @@ message IdentityGate {
 message HPowGate {
   FloatArg exponent = 1;
 }
+
+message ResetGate {
+    string reset_type = 1;
```

Comment:
`reset_type` is not yet used.  Please add comment describing the purpose of this field.

Reply:
Done.